docs/source/getting_started/projects_and_users.rst

```diff
@@ -7,6 +7,11 @@ Overview
 
 Projects are containers for other resources, such as datasets and modules.
 Access rights on resources inside a Project are managed, granted, and revoked at the Project level.
+This means a user can have permission to perform an action in one project, but be rejected because 
+they are trying to do it from the wrong project. For example, consider Dataset A owned by Project A. 
```

Comment:
`trying to do it from the wrong project` - I might use more specific language to clarify what "from" the wrong project means.

Reply:
> `trying to do it from the wrong project` - I might use more specific language to clarify what "from" the wrong project means.

Ok I tried to use more helpful language. Let me know if you have any more suggestions.